code/web/src/modules/user/EditProfile.js

```diff
@@ -8,51 +14,59 @@ class EditProfile extends Component {
         super()
         this.state = {
             //image: ?
-            name: '',
-            description: '',
-            email: '',
-            address: ''
+            name: props.user.details.name || '',
+            description: props.user.details.description || '',
+            email: props.user.details.email || '',
+            address: props.user.details.address || ''
         }
```

Comment:
Nice work on this. I like that the user will be able to choose what to edit without having to edit the entire user profile.